__mocks__/fs.ts

```diff
@@ -0,0 +1,5 @@
+const fs = jest.genMockFromModule("fs") as any;
```

Comment:
Implement correct type

Reply:
we have `@types/node` which should include `fs` (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/fs.d.ts#L1595). what was the error you were getting here?

Reply:
I tried:

```
import * as fsModule from "fs";

const fs = jest.genMockFromModule<fsModule>("fs");

fs.existsSync = () => true;
 
module.exports = fs;

```

But that gives me:

![image](https://user-images.githubusercontent.com/151311/67902403-7969d080-fb3f-11e9-9f74-eeb8c75ef04d.png)

Doing some searching and found, that that is expected. Not really sure how to type this? 🤔

Reply:
I saw this on S/O: https://stackoverflow.com/questions/52113681/how-can-i-mock-a-third-party-node-module-function-using-jest-with-typescript

Looks like they're doing slightly the same thing.